### ISSUE TITLE
Support chalk in output

### ISSUE THREAD

**Issue by kentcdodds** [CompuIves/codesandbox-client#788]
If you check out the failing test here: https://codesandbox.io/s/7ko213ovzj

You'll see:

<img width="641" alt="screen shot 2018-05-02 at 8 37 10 pm" src="https://user-images.githubusercontent.com/1500684/39558014-b363ed3e-4e48-11e8-9d3f-4ffb59a73dc3.png">

As you can see, the output is using whatever magic they use to make colors work in the terminal, but it's not colored in the output. I'm pretty sure the underlying package is `chalk` and I feel like there's a package out there for making chalk highlights work in the browser but I'm not 100% certain of that. In any case it'd be great to either figure out a way to disable chalk in the browser _or_ enable it in test output :)

**Comment by nitin42** [comment]
👋

I am gonna start looking into this issue and will ping you soon!

**Comment by nitin42** [comment]
Is this an improvement ? 
![screenshot 2018-07-20 at 9 14 24 pm](https://user-images.githubusercontent.com/16336071/43011909-81eaad44-8c62-11e8-8460-f5a551cb637f.png)

> The right one is CodeSandbox terminal and the left one is actual terminal. You can compare the rendered text.

Parsing chalked text here is complex. The only choice for us is to disable it in browser which is impossible considering how `chalk` is built. But I did try converting ANSI text to html in order to improve the output using [`ansi-html`](https://github.com/Tjatse/ansi-html), and ended up with this.  Let me know!

**Comment by kentcdodds** [comment]
Hmmm... Yeah, I guess that's probably better, but still not great. Sure would be cool if it could just be rendered the same, but I understand if that's technically infeasible. I'm going to try to disable chalk when running in codesandbox somehow. Will give it more thought. Thanks for working on this.

**Comment by nitin42** [comment]
Sadly! May be try this https://github.com/chalk/chalk#chalksupportscolor

**Comment by CompuIves** [comment]
Okay, so this one was a bit confusing. The `div` in the output doesn't show because they are rendered to HTML, which is **extremely** bad. I already escaped HTML for known validation errors, but not for custom errors.

That's why I quickly created a commit to escape this HTML too, in the process I added the `ansi-html` part. Commit can be found here (https://github.com/CompuIves/codesandbox-client/commit/0ed9d0b9d7dc166e42e42f001a96cc51ac121f3f).

Sorry for the sweepin on this on @nitin42, I wanted to get rid of this vector of attack as soon as possible!

This is what I used to test: https://codesandbox.io/s/q29pnjypj
